modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-image-set-examples_{context}"]
+= Image set configuration examples
+
+Review the following examples of `ImageSetConfiguration` files for various use cases.
+
+[discrete]
+[id="oc-mirror-image-set-examples-helm_{context}"]
+== Include arbitrary images and helm charts
+
+The following `ImageSetConfiguration` file includes a UBI image and helm charts.
+// TODO: get more specific?
```

Comment:
@afflom @jpower432 How does this first example file look, and is this description okay?

Reply:
The example looks great. Only nit is that the example description does not include a reference to the `additionalImages` configuration (it only references helm).

Reply:
Thanks, I updated the descriptions a bit after I added the second example. PTAL to and let me know what you think now